cwf/gateway/go.mod

```diff
@@ -44,7 +44,7 @@ require (
 	github.com/stretchr/testify v1.7.0
 	github.com/vishvananda/netlink v1.1.0
 	golang.org/x/crypto v0.0.0-20210921155107-089bfa567519
-	google.golang.org/grpc v1.36.0
```

Comment:
Is this change intentional? (not a go expert here)

Reply:
Good question Nils. I would think this change is not desired (unless strictly necessary for the PR). @christinewang5 ?